addons/mass_mailing/static/src/snippets/s_alert/000.scss

```diff
@@ -17,15 +17,15 @@
     }
     .s_alert_sm {
         padding: $grid-gutter-width/3;
-        font-size: $font-size-sm;
+        @include font-size($font-size-sm);
```

Comment:
I don't think it make sense to target the mass_mailing application. I don't think they handle media query and it's not the frontend anyway, let's remove this.